Client/Frontend/Browser/GridTabViewController.swift

```diff
@@ -574,8 +574,9 @@ extension GridTabViewController {
         }
 
         let controller = AlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        // Ecosia: Change close all tabs button style
```

Comment:
Thanks for the `// Ecosia` styled comment 🙌 
I'd add some quotes to highlight the copy like the following:
```suggestion
        // Ecosia: Change the "Close all" tabs button style
```
WDYT?

Reply:
I thought about it, but didn't want the comment to be "linked" with the button text in case it changes and instead link it to the functionality 🤔
Do you think that makes sense or would you still add the quotes?